base/templates/base/blocks/duo_img.html

```diff
@@ -4,7 +4,7 @@
 <div class="col-xs-12 duo-wrapper">
     <div class="col-xs-12 col-sm-6 collex-duo">
         <figure>
-            <a href="{{original_one.url}}" data-toggle="lightbox"><img class="img-responsive" src="{{original_one.url}}"></a>
+            <a href="{{original_one.url}}" data-toggle="lightbox"><img class="img-responsive" src="{{original_one.url}}" alt="{% firstof value.alt_text value.image_one.alt_text %}"></a>
```

Comment:
I think it might be impossible to populate this with `value.alt_text` or maybe I just forgot how to do it. If I'm right, we should just use `value.image_one.alt_text` and abandon the `firstof` tag. Does that make sense? Let me know if I just forgot how to do this.

Reply:
Without the `firstof` the site blows up. I copied this structure from the solo image template, which is using `firstof`. Let me know if there is a different way I would need to structure this if `firstof` is problematic.

Reply:
Additionally: removing `value.alt_text` populated the alt text area with weird code instead of the actual text in the data field. It was bizarre.

Reply:
Without `firstof` you would just do it like this `alt="{{value.image_one.alt_text}}"`, I think (notice `{{` instead of `{%`). I know when I've done the `firstof` technique in the past, I was throwing values at it that were populating from different fields in order of preference. I just can't for the life of me figure out how to populate `value.alt_text` and even if I could, I think it would be the same for both images. In order to see this concretely, I just dumped their values at the top of the template in `p` tags along with those for `value.image_one.alt_text` (and `image_two`) just so I could see them when I tried populating them in the admin. You might be getting an error because you did something like this `{% value.image_one.alt_text %}`. That's my guess.

Reply:
This has been updated to `alt="{{value.image_one.alt_text}}`